composer.json

```diff
@@ -33,12 +33,10 @@
         "thesis/endian": "^0.1.0"
     },
     "require-dev": {
-        "bamarni/composer-bin-plugin": "^1.8",
-        "ergebnis/composer-normalize": "^2.45",
-        "phpunit/phpunit": "^11.5",
-        "symfony/var-dumper": "^7.2"
+        "bamarni/composer-bin-plugin": "^1.8.2",
+        "phpunit/phpunit": "^12.1.4",
+        "symfony/var-dumper": "^6.4.15 || ^7.2.6"
     },
-    "minimum-stability": "stable",
```

Comment:
This is the default in composer.json. https://getcomposer.org/doc/04-schema.md#minimum-stability